src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/calendar/NextHearingDateTypeConfiguratorTest.java

```diff
@@ -1173,4 +1173,36 @@ public void shouldNotRecalculateDateWhenDueDateIsUnconfigurableButOriginDuration
         assertThat(configurationDmnEvaluationResponses).isEmpty();
     }
 
+    @Test
+    public void shouldReturnEmptyNextHearingDate() {
```

Comment:
minor comment - we prefer `snake_case` for test names although I accept there's a mix in the code and this file has `camelCase`.  It's ok if you leave it as it is and can refactor later.